tests/unit/test_cube.py

```diff
@@ -392,6 +400,10 @@ def but_it_raises_on_other_cube_response_types(self, cube_response, expected_val
 
     # fixture components ---------------------------------------------
 
+    @pytest.fixture
+    def _cube_dict_prop_(self, request):
```

Comment:
is this fixture component mandatory considering that it's used only in `it_knows_its_title` test? IMHO I would mock that property in the test directly

Reply:
Not at the moment, but that's because the unit test coverage is still very spotty on this class. I made the judgement to include it since that property is used in like four other methods so will be needed then.

Reply:
👍